### ISSUE TITLE
math mode preview tilde over letters not showing

### ISSUE THREAD

**Issue by phdoerfler** [SublimeText/LaTeXTools#996]
Minimal example:

```
\begin{displaymath}
A\tilde{B}C
\end{displaymath}
```

Displays as:

![bildschirmfoto 2017-01-08 um 12 37 25](https://cloud.githubusercontent.com/assets/163428/21749489/77bf7a46-d59f-11e6-9973-87b78285d97b.png)

Should resemble something like this (Skim):

![bildschirmfoto 2017-01-08 um 12 39 50](https://cloud.githubusercontent.com/assets/163428/21749493/9a883932-d59f-11e6-8263-1997797d619e.png)

As to my understanding `\tilde` does not need any extra package, but just in case this is how my .tex starts:

```
\documentclass[a4paper, 9pt, twocolumn]{extarticle}
\usepackage{amsmath,graphicx,amssymb,cite}
\usepackage[utf8]{inputenc}
\usepackage[english]{babel}

\usepackage{url}
\usepackage{hyperref} % For \autoref
\usepackage{mathrsfs} % For script letters in math mode
\usepackage{mathtools} % For the assignment operator \coloneqq which replaces ":="
```

PS: I _love_ the equation previews, thanks for your time!

**Comment by r-stein** [comment]
Ah ok it seems, that `displaymath` does not support stars, i.e. `displaymath*` does not work correctly. Just change [this setting](https://github.com/SublimeText/LaTeXTools/blob/v3.12.13/LaTeXTools.sublime-settings#L142) to `"preview_math_no_star_envs": ["displaymath"],`.
I will push the corresponding fix to the master.

**Comment by r-stein** [comment]
PS. The fix will only available in the next release so add it to your settings until then.

**Comment by phdoerfler** [comment]
Works for me, thanks a lot!

**Comment by r-stein** [comment]
Fixed in v3.12.14
